lib/std/os.zig

```diff
@@ -3327,3 +3327,45 @@ pub fn getrusage(who: i32) rusage {
         else => unreachable,
     }
 }
+
+pub const TermiosError = error {
+    InvalidFileDescriptor,
+    NotATerminal,
+    InvalidAction,
+    InterruptedBySignal,
+    ProcessOrphaned,
+};
+
+pub fn tcgetattr(handle: fd_t) TermiosError!termios {
+    if (builtin.os == .linux) {
```

Comment:
Get rid of this check; and just let the compiler error out on other platforms when `tcgetattr` hasn't been implemented